File: src/Exercise14.py
import array as arr
fibonacci_numbers = arr.array("i", [0, 1])
while len(fibonacci_numbers) <= 10:
    next_number = fibonacci_numbers[-1] + fibonacci_numbers[-2]
    fibonacci_numbers.append(next_number)

print(fibonacci_numbers)

#using an array to print the first 10 fibonacci numbers

File: Randy-JRA/Exercise_14.cpp
//Challenge 14 is to Print the first 10 Fibonacci numbers
//without using recursion.
//Reminder:
//F(0) = 0
//F(1) = 1
//F(n) = F(n-1) + F(n-2)

#include <iostream>
using namespace std;

#include <iostream>
using namespace std;

int fib(int n) {
    if (n <= 1)
        return n;
    return fib(n - 1) + fib(n - 2);
}

int main() {
    for (int i = 0; i < 10; i++)
        cout << fib(i) << " ";
    return 0;
}
